src/source/PeerConnection/Rtcp.c

```diff
@@ -25,3 +30,43 @@ STATUS onRtcpPacket(PKvsPeerConnection pKvsPeerConnection, PBYTE pBuff, UINT32 b
 
     return retStatus;
 }
+
+STATUS handleRemb(PRtcpPacket pRtcpPacket, PKvsPeerConnection pKvsPeerConnection)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+    PUINT32 pSsrcList = NULL;
+    DOUBLE maximumBitRate = 0;
+    UINT8 ssrcListLen;
+    UINT32 i;
+    PDoubleListNode pCurNode = NULL;
+    PKvsRtpTransceiver pTransceiver = NULL;
+    UINT64 item;
+
+    CHK_STATUS(rembValueGet(pRtcpPacket->payload, pRtcpPacket->payloadLength, &maximumBitRate, NULL, &ssrcListLen));
+    CHK(NULL != (pSsrcList = (PUINT32) MEMCALLOC(1, SIZEOF(UINT32) * ssrcListLen)), STATUS_STORE_OUT_OF_MEMORY);
```

Comment:
How big do you expect the buffer to be? If this is anything less than 100K you can simply use stack buffer without dynamic alloc

Reply:
`255 * uint32_t`

Reply:
You ok with me doing a `uint32_t ssrcList[255] = {}`?